requirements.txt

```diff
@@ -1,9 +1,34 @@
-torch
```

Comment:
could you explain the changes here? this seems to be listing transitive dependencies

Reply:
Updated my PR.

Ideally the `setup.py` should be updates to define the minimum requirement of `torch`. The pinning of torch version is to resolve potential issues such as the imports of `LRScheduler` or other similar cases this PR resolves the issue atm.

Other reasons to pin the torch version is with compatibility of torch versions. Here is an example:

> Prior to PyTorch 1.1.0, the learning rate scheduler was expected to be called before the optimizer’s update; 1.1.0 changed this behavior in a BC-breaking way. If you use the learning rate scheduler (calling `scheduler.step()`) before the optimizer’s update (calling `optimizer.step()`), this will skip the first value of the learning rate schedule. If you are unable to reproduce results after upgrading to PyTorch 1.1.0, please check if you are calling `scheduler.step()` at the wrong time.

Found here, https://pytorch.org/docs/stable/optim.html 

This means that even the imports could work for torch `1.1.0` there could be potential issues on what version of torch the `AutoUnit` was written with.

Could address in separate PR.

Reply:
Thanks, I agree we should address this in a separate PR. Currently the CI tests against the nightly version. We will clarify which PyTorch versions we expect TorchTNT to be compatible with.